build_projects/dotnet-cli-build/CompileTargets.cs

```diff
@@ -270,9 +274,30 @@ private static void RemovePdbsFromDir(string directory)
                 GenerateNuGetPackagesArchive(c, dotnet, sdkOutputDirectory);
             }
 
+            CopyMSBuildTargetsToSDKRoot(sdkOutputDirectory);
+
             return c.Success();
         }
 
+        private static void CopyMSBuildTargetsToSDKRoot(string sdkOutputDirectory)
+        {
+            var msbuildTargetsDirectory = Path.Combine(sdkOutputDirectory, "runtimes", "any", "native");
```

Comment:
Not sure I follow.  Is this a "point in time" thing?  Or do we expect this is how these packages will be for RTM?